src/trigrams.py

```diff
@@ -5,12 +5,12 @@
 
 
 def parse(text):
-    """Turns text into a list of words."""
+    """Turn text into a list of words."""
     return text.split()
 
 
 def create_kvp(text_list):
-    """Turns text_list into a dictionary of key value pairs in which every two consecutive words is a key for the value of the following word."""
+    """Turn text_list into a dictionary of key value pairs in which every two consecutive words is a key for the value of the following word."""
     dct = {}
     for idx in range(len(text_list) - 2):
         key = text_list[idx] + "_" + text_list[idx + 1]
```

Comment:
Interesting approach to use a underscore to create a key. This will probably cause more work for your program to do later on.

You should absolutely very much look at the documentation and some examples for the `str.join()` method.